### PR TITLE
ani-cli termux compatibility

### PR DESCRIPTION
**Pull request by John-Russel**
I made it much simpler this time :)

i deleted the previous one because i dont know how to reset it 

if you want to merge it
you can edit the line in README.md
> git clone https://github.com/John-Russel/ani-cli.git
`to`
> git clone https://github.com/pystardust/ani-cli.git

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,3 +13,22 @@ uninstall:
...
+	# creating mpv file
+	@echo 'am start --user 0 -a android.intent.action.VIEW -d "$$2" -e "http-header-fields" "$$1" -n is.xyz.mpv/.MPVActivity' > $(TERMUX_BIN)/mpv
+	# marks mpv executable
+	chmod +x $(TERMUX_BIN)/mpv
+	# creating .cache folder
+	mkdir -p $(HOME)/.cache
+
+uninstall-termux-mpv:
+	rm -rf $(TERMUX_BIN)/ani-cli
+	rm -rf $(TERMUX_BIN)/mpv
```
Comment (ura43) [inline Makefile:34]: Why should this be in master?
Just implement the modified make file in your fork, as referenced in your Readme changes

**Review by ura43 — CHANGES_REQUESTED** [review]
We're getting close

File: Makefile (hunk 1 above)
Comment (John-Russel) [reply Makefile:34]: it has a error if i use the `make` 
Screenshot_2022-01-18-21-04-39-62
and if I change the PREFIX to fix the error the other os may have the same error so I just added a new line for termux installation

Comment (John-Russel) [reply Makefile:34]: oh sorry i not good in english

you can just add the termux installation in README.md
you don't need to add this changes
only add this changes if you'd like to change the git clone in README.md

Comment (CoolnsX) [reply Makefile:34]: > Why should this be in master? Just implement the modified make file in your fork, as referenced in your Readme changes

I think he doesn't know about creating and managing branch.. so @ura43 can you create seperate android branch and push it??

Comment (ura43) [reply Makefile:34]: Sure @CoolnsX I hope I get to it sometime today

**Pull request closed without merging**